syntax_transform/ClassNew.cc

```diff
@@ -1,16 +1,16 @@
-#include "dsl/ClassNew.h"
+#include "syntax_transform/ClassNew.h"
 #include "ast/Helpers.h"
 #include "ast/ast.h"
 #include "core/Context.h"
 #include "core/Names.h"
 #include "core/core.h"
-#include "dsl/dsl.h"
+#include "syntax_transform/syntax_transform.h"
 
 using namespace std;
 
-namespace sorbet::dsl {
+namespace sorbet::syntax_transform {
 
-vector<unique_ptr<ast::Expression>> ClassNew::replaceDSL(core::MutableContext ctx, ast::Assign *asgn) {
+vector<unique_ptr<ast::Expression>> ClassNew::run(core::MutableContext ctx, ast::Assign *asgn) {
```

Comment:
I liked that we could distinguish between different kinds of DSL passes before. patchDSL / replaceDSL are not both conceptually `run` in my mind. I'm in general much happier writing `replaceDSL` DSL passes than `patchDSL` passes when possible.

Reply:
sure, I can keep the distinction. What would you name them in the new world?